scripts/e2e.sh

```diff
@@ -7,14 +7,16 @@ setup_service google-chrome-stable 89.0.4389.72-1
 
 export TEST_SUITE_TYPE="junit"
 export TEST_RESULT_FILE_DIR="${REPO}/reports/e2e"
+echo ${TEST_SUITE_TYPE} > ${TEST_SUITE_TYPE_FILE}
+echo ${TEST_RESULT_FILE_DIR} > ${TEST_RESULT_FILE_DIR_FILE}
```

Comment:
We weren't setting these until the end, which mean that if the script failed in the middle (like on line 25), Bacon wouldn't save and export any output to be displayed (it ignores any files that aren't in the specified output directory)